functions.php

```diff
@@ -380,45 +380,71 @@ function nightingale_register_required_plugins() {
+	 * N.B - You must set formidaable forms to 'Do not use formidable styles' for it to kick in
+	 */
+	if ( in_array( 'formidable/formidable.php', $active_plugins, true ) ) {
+		if ( ! is_admin() ) {
+			require get_template_directory() . '/inc/formidable.php';
+		}
 	}
-}
 
+	/**
+	 * LearnDash style over-ride.
+	 * N.B. This is not a plugin, nor does it provide any plugin-like changes. This is a theme file for
+	 * the LearnDash plugin so any content generated by LearnDash fits in to this theme.
+	 * The check around the require is to see if the plugin is active on this install
+	 */
+	if ( in_array( 'sfwd-lms/sfwd_lms.php', $active_plugins, true ) ) {
+		if ( ! is_admin() ) {
+			require get_template_directory() . '/inc/learndash.php';
+		}
 
-/**
- * Formidable Forms style over-ride.
- * N.B. This is not a plugin, nor does it provide any plugin-like changes. This is a theme file for
- * the Formidable Forms plugin so any content generated by Formidable Forms fits in to this theme.
- * The check around the require is to see if the plugin is active on this install
- * N.B - You must set formidaable forms to 'Do not use formidable styles' for it to kick in
- */
-if ( in_array( 'formidable/formidable.php', $active_plugins, true ) ) {
-	if ( ! is_admin() ) {
-		require get_template_directory() . '/inc/formidable.php';
+		add_action( 'admin_head', 'nightingale_learndash_admin_fix' );
 	}
-}
 
-/**
- * LearnDash style over-ride.
- * N.B. This is not a plugin, nor does it provide any plugin-like changes. This is a theme file for
- * the LearnDash plugin so any content generated by LearnDash fits in to this theme.
- * The check around the require is to see if the plugin is active on this install
- */
-if ( in_array( 'sfwd-lms/sfwd_lms.php', $active_plugins, true ) ) {
-	if ( ! is_admin() ) {
-		require get_template_directory() . '/inc/learndash.php';
+	/**
+	 * Events Calendar style over-ride.
+	 * N.B. This is not a plugin, nor does it provide any plugin-like changes. This is a theme file for
+	 * the Events Calendar plugin so any content generated by Events Calendar fits in to this theme.
+	 * The check around the require is to see if the plugin is active on this install
+	 */
+	if ( in_array( 'the-events-calendar/the-events-calendar.php', $active_plugins, true ) ) {
+		if ( ! is_admin() ) {
+			require get_template_directory() . '/inc/events-calendar.php';
+		}
+	}
+
+	/**
+	 * Cookie Control plugin check.
+	 * N.B. This is not a plugin, nor does it provide any plugin-like changes. This is a theme file for
+	 * the Cookie Control plugin so the element is presented first on screen for improved accessibility
+	 * on screen readers and for keyboard navigation.
+	 * The check around the require is to see if the plugin is active on this install
+	 */
+	if ( in_array( 'cookie-notice/cookie-notice.php', $active_plugins, true ) ) {
+		require get_template_directory() . '/inc/cookie-notice.php';
 	}
```

Comment:
PHPCS for WP says ~ Blank line found after control structurePHPCS(WordPress.WhiteSpace.ControlStructureSpacing.BlankLineAfterEnd)

Reply:
Fixed (I think)